src/components/FullTeam/FullTeam.vue

```diff
@@ -1,7 +1,30 @@
 <template>
-  <div class="team">
-    <div v-for="(member, id) in shuffle(members)" v-bind:key="id">
-      <TeamMember class="member" :member="member" />
+  <div class="wrapper">
+    <div
+      class="rect"
+      id="rect"
+      @mouseover="showFire = true"
+      @mouseleave="showFire = false"
+    >
+      <transition name="fadeFire">
+        <img v-show="showFire" class="campfire" src="@/assets/campfire.gif" />
+      </transition>
+    </div>
+    <div
+      class="rectTent"
+      id="rectTent"
+      @mouseover="showTent = true"
+      @mouseleave="showTent = false"
+    >
+      <transition name="fadetent">
+        <img v-show="showTent" class="camptent" src="@/assets/tent.webp" />
+      </transition>
+    </div>
+
+    <div class="team" id="team">
+      <div v-for="(member, id) in shuffle(members)" v-bind:key="id">
+        <TeamMember v-show="showFullTeam" class="member" :member="member" />
```

Comment:
Team page re-renders and re-orders every time the fire is shown. Can we make the easter-egg a bit less destructive? :D

Reply:
Are there any other requests than the destructive animation? :D I have deleted them for now and will discuss the case in the group - tbh its not my wanted solution